src/pages/Login.jsx

```diff
@@ -97,7 +97,7 @@ export default function Login() {
           />
         }
       ></Input>
-      <Link href="/resetpassword">Forgot Password?</Link>
+      <Link href="/reset-password">Forgot Password?</Link>
```

Comment:
(praise): Awesome. Using kebab case for routes with multiple words really helps with readability.